src/configure.py

```diff
@@ -439,7 +438,7 @@ def main(args):
     )
 
     # 1. find where configuration is
-    config_dir = os.path.expanduser(args.config_dir)
+    config_dir = os.path.expanduser(os.path.join(args.reports_base + CONFIG_DIR))
```

Comment:
Should it not use BASE_DIR constant if there is no argument for report dir?

Reply:
no, no. It was actually over defined before. Look at launch_common.py line 209 and "--reports_base"